docs/user/reference/general/experiments-api.md

```diff
@@ -305,8 +319,61 @@ Assert.equals(
 
 {{#include ../../../shared/tab_footer.md}}
 
+### `testGetExperimentationId`
+
+Returns the current Experimentation ID, if any.
+
+{{#include ../../../shared/tab_header.md}}
+
+<div data-lang="Kotlin" class="tab">
+
+```Kotlin
+assertEquals("alpha-beta-gamma-delta", Glean.testGetExperimentationId())
+```
+</div>
+
+<div data-lang="Java" class="tab"></div>
+
+<div data-lang="Swift" class="tab">
+
+```Swift
+XCTAssertEqual(
+  "alpha-beta-gamma-delta",
+  Glean.shared.testGetExperimentationId()!,
+  "Experimenatation ids must match"
+)
+```
+</div>
+
+<div data-lang="Python" class="tab">
+
+```Python
+from glean import Glean
+
+assert "alpha-beta-gamma-delta" == Glean.test_get_experimentation_id()
+```
+</div>
+
+<div data-lang="JavaScript" class="tab" data-bug="1850323"></div>
+
+<div data-lang="Rust" class="tab">
+
+```Rust
+assert_eq!(
+  "alpha-beta-gamma-delta".to_string(),
+  glean_test_get_experimentation_id(),
+  "Experimentation id must match"
+);
+```
+</div>
+
+<div data-lang="Firefox Desktop" class="tab" data-bug="1850479"></div>
+
+{{#include ../../../shared/tab_footer.md}}
+
 ## Reference
 
 * [Swift API docs](../../../swift/Classes/QuantityMetricType.html)
 * [Python API docs](../../../python/glean/metrics/quantity.html)
 * [Rust API docs](../../../docs/glean/private/quantity/struct.QuantityMetric.html)
+* [String Metric Type](../metrics/string.md)
```

Comment:
Updated to use the relative path to the .md files? I'm not sure I understand what you are asking for.

Reply:
woops, tihs should have marked all 4 lines. The other links go to the quantity metric.

Reply:
There are other places in the docs that use the .html linking. Is this worth filing a bug to update all of them?

Reply:
I intended to mark the linkes 376-379 here insteaf of only 379.

There's 4 links here: Swift API docs, Python API docs, Rust API docs, String Metric Type.
The first 3 links in this document link to the respective API docs for the quantity metric. 
But this is the document for the experimentation ID.
Those links need to change to the respective API docs for `set_experimentation_id`.
They always end in `.html`, as there is no `.md` file our tool (`mdbook`) knows about.

The last link is to the string metric. That's an "internal" link: a link to a different page in this book.
I initially considered it wrong here, but given we mention string metric type in the docs a couple of times I think we could leave this in.

Reply:
Ahh, I see. Yeah, since I linked to the specific StringMetric things directly, I don't think it needs to stay. I'll update the others to point to the correct pages.